lib/src/main/java/ECSE428Project/model/Round.java

```diff
@@ -0,0 +1,70 @@
+package ECSE428Project.model;
```

Comment:
there's round and match?

Reply:
Yes a match is composed of multiple rounds

Reply:
I just followed what was in the UML Class diagram we might want to change the terms later on but I don't think it matters too much